src/flake8_pytest_style/plugins/unittest_assert.rs

```diff
@@ -23,11 +25,12 @@ pub enum UnittestAssert {
     IsNone,
     IsNot,
     IsNotNone,
-    ItemsEqual,
     Less,
     LessEqual,
     ListEqual,
+    Logs,
```

Comment:
Maybe we should exluce `Logs` and `NoLogs` because they can be simply replaced with `assert ...`.

Reply:
The same applies to `Warns` and `Raises`.

Reply:
For `Warns` and `Raises` (and their variants), we could say `use pytest.raises|pytest.warns` instead of `use assert ...`.